install/kubernetes/helm/istio/charts/mixer/templates/deployment.yaml

```diff
@@ -17,7 +17,7 @@
         projected:
           sources:
           - serviceAccountToken:
-              audience: {{ $.Values.global.trustDomain }}
+              audience: {{ $.Values.global.sds.token.aud }}
```

Comment:
does it make sense to rename this to `tokenAudience` instead of `token.aud`? Or `tokenAud`.. just not sure we need more nesting?

Reply:
Later we may want to have `exp` for the expiration, so I think it's better to have them nested under `token`.